server/controllers/auth/login.js

```diff
@@ -3,35 +3,61 @@ const { sign } = require('jsonwebtoken');
 
 const loginValidation = require('../../utils/validation/loginValidation');
 const applicant = require('../../database/models/applicant');
+const admin = require('../../database/models/admin');
+
 require('env2')('config.env');
 
+const errorResponse = {
+  status: 'failed',
+  type: 'Wrong Email or Password',
+  errorMessage: 'Check Email or password',
+};
+
 const login = (req, res) => {
   const { email, password } = req.body;
   loginValidation(email, password)
     .then((valid) => {
-      if (!valid) res.status(400).json({ error: 'Check the data you entered' });
+      if (!valid)
+        res.status(400).json({
+          status: 'failed',
+          type: 'type error',
+          errorMessage: 'Check the data you entered',
+        });
       else {
         applicant
           .findOne({ email })
           .then((data) => {
             bcrypt.compare(password, data.password, (err, result) => {
```

Comment:
@ahmadysalah you used bcrypt without requiring it in top file

Reply:
not you can't, but this file doesn't have changed in rows 1 to 4 look again.

Reply:
ok, I didn't see it